playbooks/precis.yml

```diff
@@ -6,3 +6,23 @@
 - hosts: precis
   roles:
     - precis
+  tasks:
+    - name: Configure virtual host.
+      include_role:
+        name: apache
+        tasks_from: host
+      vars:
+        vip_address: "{{ precis_vip_address }}"
+        apache_server_priority: 25
+        apache_server_alias: precis_public
+        apache_server_name: "{{ precis_server_hostname }}"
+        apache_ssl_letsencrypt_enabled: no
+        apache_server_https_port: "{{ precis_endpoint_port }}"
+        apache_server_http_port: null
+        apache_server_ssl_certificate_file: "{{ precis_ssl_file_prefix }}_chameleoncloud_org_cert.cer"
```

Comment:
I generally believe that making these things explicit is a better place to start, especially when the pattern is not yet well-formed (we have 2 examples, maybe we can revisit when we know what TACC / other sites are doing). I suggest `precis_ssl_certificate_file` having the full file name.